apps/data-extraction/src/main.ts

```diff
@@ -123,7 +127,7 @@ const operationsSelection = new MultiSelect({
     'everything',
     ...extractors.map(extractor => {
       return extractor.getName();
-    })
+    }).sort((a, b) => a.localeCompare(b)),
```

Comment:
The extractors are listed in order to avoid having issues with extractor dependencies. The mappy and LGB extractors require informations from previous extractors, which is why the array isn't sorted programatically, it's made by hand.

Reply:
Sorry, I didn't know :/